testui/elements/testui_element.py

```diff
@@ -205,41 +208,59 @@ def visible_for(self, seconds=1):
                            f'for {seconds}s')
         return self
 
-    def wait_until_visible(self, seconds=10.0, log=True):
+    def wait_until_visible(self, seconds=10.0, log=True) -> 'Elements':
         start = time.time()
+        
         err = None
+
         err_text = 'not'
         if self.__is_not:
             err_text = 'is'
+        
         i = 0
         while time.time() < start + seconds or i < 1:
             try:
-                self.get_element()
-                if not self.__is_not:
+                is_visible = self.get_element().is_displayed()
```

Comment:
why not reuse `is_visible` function with `log=False` instead of duplicating logic.

Reply:
I think there's no need for it tbh